src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -29,10 +32,14 @@
     private final FilteredList<EventTask> filteredEventTasks;
     private final FilteredList<DeadlineTask> filteredDeadlineTasks;
 
+    //for undo
+    private ArrayList<Commit> commits = new ArrayList<Commit>();//a stack of past TaskBook states
+    private int head;
```

Comment:
There should be a comment explaining what `head` is and what kinds of values it has.